modules/data-entry/src/main/frontend/src/questionnaire/Subject.jsx

```diff
@@ -120,7 +122,15 @@ function Subject(props) {
     <React.Fragment>
       <NewFormDialog currentSubject={currentSubject} withButton buttonTitle={ "New questionnaire for this " + (currentSubject?.type?.label || "Subject") } />
       <Grid container spacing={4} direction="column" className={classes.subjectContainer}>
-        <SubjectHeader id={currentSubjectId} key={"SubjectHeader"} pageTitle={pageTitle} classes={classes} getSubject={handleSubject} history={history} contentOffset={props.contentOffset}/>
+        <SubjectHeader
+          id={currentSubjectId}
+          key="SubjectHeader"
+          pageTitle={pageTitle}
+          classes={classes}
+          getSubject={handleSubject}
+          reloadSubject={fetchRelatedRef}
+          history={history}
+          contentOffset={props.contentOffset}/>
```

Comment:
Very minor code style suggestion:
```suggestion
          contentOffset={props.contentOffset}
          />
```